lib/src/models/nodes.ts

```diff
@@ -73,3 +74,34 @@ export interface BodyNode {
   description?: Locatable<string>;
   type: DataType;
 }
+
+export interface TestNode {
+  description?: Locatable<string>;
+  states?: TestStateNode[];
```

Comment:
ditto

Reply:
Planning to introduce `Locatable` to some of the `@test` related nodes in the future